launch/model_endpoint.py

```diff
@@ -199,11 +199,18 @@ def __init__(self, client, endpoint_name: str, async_task_id: str):
         self.endpoint_name = endpoint_name
         self.async_task_id = async_task_id
 
-    def get(self) -> EndpointResponse:
+    def get(self, timeout: float = 0) -> EndpointResponse:
```

Comment:
nit: should this be `timeout: float = 0.0`?

Reply:
Just used `None`